libminifi/src/SchedulingAgent.cpp

```diff
@@ -115,11 +115,11 @@ bool SchedulingAgent::onTrigger(core::Processor* processor, const std::shared_pt
     processor->onTrigger(processContext, sessionFactory);
     processor->decrementActiveTask();
   } catch (std::exception &exception) {
-    logger_->log_debug("Caught Exception %s", exception.what());
+    logger_->log_debug("Caught Exception during SchedulingAgent::onTrigger, type: %s, what: %s", typeid(exception).name(), exception.what());
```

Comment:
I think this log msg would be more talkative in case we log the name of the processor and the uuid of that to identify which processor threw the given exception.